Python/robodk/robolink.py

```diff
@@ -5201,16 +5204,23 @@ def setRobot(self, robot=None):
         :param robot: robot to link
         :type robot: :class:`.Item`
         """
-        with self.link._lock:
-            if robot is None:
-                robot = Item(self.link)
-            self.link._check_connection()
-            command = 'S_Robot'
-            self.link._send_line(command)
-            self.link._send_item(self)
-            self.link._send_item(robot)
-            self.link._check_status()
+        if self.link.BUILD > 22916:
+            self.setLink(robot)
```

Comment:
What is the behavioural change with the old vc. new setRobot? Is it worth documenting?

Reply:
They have both the same behaviour. This change is to simplify the commands RoboDK accepts in the future.